tools/ci_build/github/apple/assemble_ios_packaging_artifacts.sh

```diff
@@ -21,12 +21,12 @@ ORT_POD_VERSION=${3:?${USAGE_TEXT}}
 SHOULD_UPLOAD_ARCHIVES=${4:?${USAGE_TEXT}}
 
 STORAGE_ACCOUNT_NAME="onnxruntimepackages"
-STORAGE_ACCOUNT_CONTAINER_NAME="ortmobilestore"  # TODO look into moving to '$web'
-STORAGE_URL_PREFIX="https://${STORAGE_ACCOUNT_NAME}.blob.core.windows.net/${STORAGE_ACCOUNT_CONTAINER_NAME}"
+STORAGE_ACCOUNT_CONTAINER_NAME='$web'
+STORAGE_URL_PREFIX="https://onnxruntimepackages.z14.web.core.windows.net"
 
```

Comment:
Is there documentation somewhere about this URL? e.g. do we need to do anything to ensure it remains valid

Reply:
found a way to not hardcode it:
https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website-how-to?tabs=azure-cli#find-the-website-url